motor/_version.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 """Version-related data for motor."""
-version_tuple = (3, 4, 0, ".dev0")
+version_tuple = (3, 3, 2, ".dev0")
```

Comment:
We should remove the `.dev0` portion.

Reply:
This should not have been removed until all the 3.3.2 commits are merged. Stilling waiting on https://github.com/mongodb/motor/pull/233 and https://github.com/mongodb/motor/pull/240 IIUC.

Reply:
(For next time). No need to change it now.